### PR TITLE
Update to Ubuntu 18.04

### PR DESCRIPTION
**Pull request by Frenzie**
Some minor prep to be finalized after 2021.03 release.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/virdevenv/57)

### CODE REVIEW

File: docker/ubuntu/baseimage/bootstrap.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ echo " ------------------------------------------"
 apt-get install -y \
     "${MISC_TOOLS[@]}" \
     build-essential dpkg-dev pkg-config python3-pip \
-    gcc-5 cpp-5 g++-5 make automake cmake ccache \
+    gcc-7 cpp-7 g++-7 make automake cmake ccache \
     ninja-build \
     patch libtool nasm autoconf2.64 \
```
Comment (Frenzie) [inline docker/ubuntu/baseimage/bootstrap.sh:33]: Probably add lua5.1 somewhere around here, cf. https://github.com/koreader/virdevenv/issues/58

**Comment by Frenzie** [comment]
@NiLuJe For some reason the branch is named git 2.18 but at the moment it includes 2.17. Do you recall any context for that? I'm not sure why I named the branch that.

**Comment by NiLuJe** [comment]
Nothing rings a bell, no, unless there was something specific you originally wanted in 2.18 that isn't in 2.17 (possibly something about shallow clones for the CI runs?).

**Comment by Frenzie** [comment]
That's certainly possible. Oh well, it's probably not very important.

**Pull request merged**